hermeto/core/errors.py

```diff
@@ -81,6 +81,40 @@ class InvalidInput(UsageError):
     """User input was invalid."""
 
 
+class UnsupportedPackageManager(InvalidInput):
+    """User requested a package manager that is not yet fully supported."""
+
+    def __init__(
+        self,
+        unsupported_managers: List[str],
+        suggested_managers: Optional[List[str]] = None,
+        *,
+        solution: Optional[str] = _argument_not_specified,
+        docs: Optional[str] = None,
```

Comment:
I would have skipped `solustion` and `docs` unless you plan to subclass this one further.

Reply:
I believe that's a copy-paste from `PackageManagerError` which served as a precedent for this.